docs/conf.py

```diff
@@ -1,10 +1,13 @@
 # -*- coding: utf-8 -*-
-#
-# Configuration file for the Sphinx documentation builder.
-#
-# This file does only contain a selection of the most common options. For a
-# full list see the documentation:
-# https://www.sphinx-doc.org/en/master/config
+
+"""Configuration file for the Sphinx documentation builder.
+
+Note
+----
+For a full list see the documentation:
+https://www.sphinx-doc.org/en/master/config
+
+"""
```

Comment:
I don't have a better place to put this.
The supported_instruments.rst isn't working, https://pysatmissions.readthedocs.io/en/rc_0_3_0/supported_instruments.html

Empty placeholders.

Reply:
```
.. image:: https://zenodo.org/badge/DOI/10.5281/zenodo.1299374.svg
   :target: https://doi.org/10.5281/zenodo.1299374
```

For the OMMBV badge in the references section.